fastparquet/test/test_converted_types.py

```diff
@@ -167,3 +167,13 @@ def test_tz_nonstring(tmpdir):
 
     round = pd.read_parquet(fn, engine="fastparquet")
     assert (event_df == round).all().all()
+
+
+def test_pandas_simple_type(tmpdir):
+    import pandas as pd
+    fn = os.path.join(tmpdir, "out.parquet")
+    df = pd.DataFrame({"a": [1, 2, 3]}, dtype='uint8')
+    df.to_parquet(fn, engine="fastparquet")
+    df2 = pd.read_parquet(fn, engine="fastparquet")
+    assert df2.a.dtype == "uint8"
```

Comment:
The values should also be checked. If you want, you can use ``assert_frame_equal`` from pandas to check all the stuff at once.

Reply:
Can do - but the dtype is what is specifically being tested here.

Reply:
nbd. thanks again for the quick patch.